bullet_train/lib/bullet_train.rb

```diff
@@ -81,7 +81,7 @@ def inbound_email_enabled?
 end
 
 def billing_enabled?
-  ENV["STRIPE_SECRET_KEY"].present? && defined?(BulletTrain::Billing)
+  (ENV["STRIPE_SECRET_KEY"].present? || ENV["PADDLE_SECRET_KEY"]) && defined?(BulletTrain::Billing)
```

Comment:
Do we need `.present?` on the new var that we're checking for?

```suggestion
  (ENV["STRIPE_SECRET_KEY"].present? || ENV["PADDLE_SECRET_KEY"].present?) && defined?(BulletTrain::Billing)
```

Reply:
Thank you @jagthedrummer that was an oversight of mine. I think eventually we'll want to refactor this and delegate to the provider gems...

Reply:
Agreed on figuring out how to delegate this to the various providers.